### PR TITLE
Fixed the name of the policy

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Fixed the name of the policy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +136,7 @@ sudo podman run --rm --name cloud-governance -e policy="ec2_idle" -e AWS_ACCESS_
 # policy=ec2_run
 sudo podman run --rm --name cloud-governance -e policy="ec2_run" -e AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID" -e AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY" -e AWS_DEFAULT_REGION="us-east-2" -e dry_run="yes" -e policy_output="s3://bucket/logs" -e log_level="INFO" "quay.io/ebattat/cloud-governance"
 
-# select policy ['ec2_stop', 's3_inactive', 'empty_roles', 'ip_unattached', 'nat_gateway_unused', 'zombie_snapshots']
+# select policy ['ec2_stop', 's3_inactive', 'empty_roles', 'ip_unattached', 'unused_nat_gateway', 'zombie_snapshots']
```
Comment (ebattat) [inline README.md:139]: Is that mean that this policy didnt run ?

Comment (athiruma) [reply README.md:139]: Yes, 😞

Comment (ebattat) [reply README.md:139]: can u check the other policies name ?

Comment (athiruma) [reply README.md:139]: ok

**Pull request merged**